### PR TITLE
Added info about modifiers #415

### LINKED ISSUE #415: Add more info about modifiers
Ascent to different behaviour in runtime.

### PR DESCRIPTION
**Pull request by godfreyd**
Closes #415

### CODE REVIEW

File: method/key-concepts/key-concepts.en.md [hunk 1]
```diff
@@ -65,6 +65,12 @@ For instance, the appearance of the menu block (`menu`) may change depending on
 
 ![Add a menu to the footer](key-concepts__site-footer-menu.png)
 
+Modifiers can be changed during block's functioning (for example, as a reaction to a DOM event of the block), or upon the request from the other blocks.
```
Comment (tadatuta) [inline method/key-concepts/key-concepts.en.md]: during block's functioning -> in runtime
or upon the request from the other blocks -> or via other blocks

File: method/key-concepts/key-concepts.en.md [hunk 2]
```diff
@@ -65,6 +65,12 @@ For instance, the appearance of the menu block (`menu`) may change depending on
...
+
+For example, if you click button Sign In (DOM event click), in case of wrong Login or Password fields, on the hidden block error messages to set the modifier (`visible`):
```
Comment (tadatuta) [inline method/key-concepts/key-concepts.en.md]: E.g. a click on Sign In button (`click` DOM event) when sign in form is filled with wrong credentials will set `visible` modifier on hidden `error` block.

File: method/key-concepts/key-concepts.ru.md [hunk 3]
```diff
@@ -65,6 +65,12 @@
 
 ![Добавить меню в подвале](key-concepts__site-footer-menu.png)
 
+Модификаторы могут изменяться как в процессе работы блока (например, как реакция на DOM-события блока), так и по запросу из других блоков.
+
+Например, при клике по кнопке Sign In (DOM-событие click), в случае неверно заполненных полей Login или Password, на скрытый блок сообщений об ошибках установить модификатор (`visible`):
+
+![Показать сообщение об ошибке](key-concepts__head.png)
```
Comment (tadatuta) [inline method/key-concepts/key-concepts.ru.md]: давай просто оставим эту часть без иллюстрации, вроде она не критична здесь

File: method/key-concepts/key-concepts.en.md [hunk 4]
```diff
@@ -65,6 +65,10 @@ For instance, the appearance of the menu block (`menu`) may change depending on
 
 ![Add a menu to the footer](key-concepts__site-footer-menu.png)
 
+Modifiers can be changed in runtime (for example, as a reaction to a DOM event of the block), or via other blocks.
+
+E.g. a click on Sign In button (click DOM event) when sign in form is filled with wrong credentials will set visible modifier on hidden error block.
```
Comment (krishnevsky) [inline method/key-concepts/key-concepts.en.md]: For example, if the wrong credentials are entered when a user clicks the Sign In button (the 'click' DOM event), the 'visible' modifier is set on a hidden block with error messages.

**Pull request merged**